napari/benchmarks/benchmark_points_layer.py

```diff
@@ -136,8 +141,7 @@ class PointsToMaskSuite:
         [5, 10],
     ]
 
-    if 'PR' in os.environ:
-        skip_params = Skiper(lambda x: x[0] > 256 or x[1][0] > 512)
+    skip_params = Skipper(func_pr=lambda x: x[0] > 256 or x[1][0] > 512)
```

Comment:
I'm having trouble parsing `func_pr=` and `func_ci=`. Maybe we can rename the class to `Skip` and the parameters `in_ci_if=` and `in_pr_if=`? That would read something like:

```
skip_params = Skip(in_pr_if=lambda shp: shp[0] > 256, in_ci_if=lambda shp: shp[0] > 512)
```

That starts to be a bit more explanatory I think. What do you think?

Reply:
I am not convinced that `in_pr_if` is more readable than `func_pr`, especially in `func_pr` you do not check the middle of the word to find which case is covered here` 

Maybe `if_in_pr`, `if_on_ci` and `always`?

Reply:
> Maybe if_in_pr, if_on_ci and always?

Sure.

I'd also rename the class to Skip. For some reason Skipper feels a bit weird — probably because it can mean [many other things](https://www.merriam-webster.com/dictionary/skipper).

Reply:
Fixed naming.